src/components/pages/Executors/ExecutorDetails/ExecutorSettings/ExecutorSettings.tsx

```diff
@@ -4,10 +4,10 @@ import {SettingsLeftNavigation, StyledSettingsContainer, StyledTabContentContain
 
 import CommandAndArguments from './CommandAndArguments';
 import ContainerImage from './ContainerImage';
-import Definition from './Definition';
+import ExecutorDefinition from './ExecutorDefinition';
 import General from './General';
 
-const tabsConfig: Array<JSX.Element> = [<General />, <ContainerImage />, <CommandAndArguments />, <Definition />];
```

Comment:
What is a point of removing type?

Reply:
The type is automatically inferred so there is no reason for typing something that is done automatically by the TypeScript compiler. 

<img width="933" alt="Screenshot 2023-02-03 at 16 47 58" src="https://user-images.githubusercontent.com/1321225/216646448-01595e62-1802-4fc0-a265-136c404ff2d0.png">